src/index.js

```diff
@@ -140,3 +137,23 @@ function parsePage($) {
 function getFileName(date) {
   return `${date.format('YYYYMM')}_free.pdf`
 }
+
+async function renameParentDirectory(fields) {
+  try {
+    // Looking for parent directory created by home
+    const importDir = await cozyClient.files.statByPath(fields.folderPath)
+    const parentDir = await cozyClient.files.statById(
```

Comment:
Aren't those calls actually using cozy-client-js ? If yes, we should not do that, new code should never rely on cozy-client-js anymore